lib/logging/log.js

```diff
@@ -0,0 +1,549 @@
+/*!
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*!
+ * @module logging/log
+ */
+
+'use strict';
+
+var arrify = require('arrify');
+var extend = require('extend');
+var format = require('string-format-obj');
+var is = require('is');
+var JSONStream = require('JSONStream');
+var nodeutil = require('util');
+var propAssign = require('prop-assign');
+var pumpify = require('pumpify');
+var streamEvents = require('stream-events');
+var through = require('through2');
+
+/**
+ * @type {module:logging/entry}
+ * @private
+ */
+var Entry = require('./entry.js');
+
+/**
+ * @type {module:common/serviceObject}
+ * @private
+ */
+var ServiceObject = require('../common/service-object.js');
+
+/**
+ * A log is a named collection of entries, each entry representing a timestamped
+ * event. Logs can be produced by Google Cloud Platform services, by third-party
+ * services, or by your applications. For example, the log `apache-access` is
+ * produced by the Apache Web Server, but the log
+ * `compute.googleapis.com/activity_log` is produced by Google Compute Engine.
+ *
+ * @resource [Introduction to Logs]{@link https://cloud.google.com/logging/docs/api/#logs}
+ *
+ * @alias module:logging/log
+ * @constructor
+ *
+ * @param {object} options - [Configuration object](#/docs/?method=gcloud).
+ *
+ * @example
+ * var gcloud = require('gcloud')({
+ *   keyFilename: '/path/to/keyfile.json',
+ *   projectId: 'grape-spaceship-123'
+ * });
+ *
+ * var logging = gcloud.logging();
+ * var log = logging.log('syslog');
+ */
+function Log(logging, name) {
+  var methods = {
```

Comment:
There are no docs yet, but it's going to be pretty much similar to V1
https://cloud.google.com/logging/docs/api/ref/rest/v1beta3/projects.logs/list
with the service replaced with monitored resource